gateway/cli/src/main.rs

```diff
@@ -92,6 +93,40 @@ pub enum Commands {
         #[clap(long)]
         network: Option<bitcoin::Network>,
     },
+    #[command(subcommand)]
+    Lightning(LightningCommands),
+}
+
+#[derive(Subcommand)]
+pub enum LightningCommands {
+    /// Connect to another lightning node
+    ConnectToPeer {
+        #[clap(long)]
+        pubkey: String,
+
+        #[clap(long)]
+        host: String,
+    },
+    /// Get a Bitcoin address to fund the gateway
+    GetFundingAddress,
+    /// Open a channel with another lightning node
+    OpenChannel {
+        /// The public key of the node to open a channel with
+        #[clap(long)]
+        pubkey: String,
+        /// The amount to fund the channel with
+        #[clap(long)]
+        channel_size_sats: u64,
+        /// The amount to push to the other side of the channel
+        #[clap(long)]
+        push_amount_sats: u64,
```

Comment:
Can we make this optional and default to 0?

Reply:
Done. I only did this for the CLI argument (i.e. we're performing `.unwrap_or(0)` here in the CLI code). I did this since I think a plain `u64` is more appropriate for `ILnRpcClient` since `None` represents the same behavior as `Some(0)`. I'm assuming this was your intention too but just wanted to lay out the reasoning here.